src/groundlight/client.py

```diff
@@ -1066,16 +1068,20 @@ def _wait_for_result(
             image_query = self._fixup_image_query(image_query)
         return image_query
 
+    # pylint: disable=duplicate-code
     def add_label(
-        self, image_query: Union[ImageQuery, str], label: Union[Label, str], rois: Union[List[ROI], str, None] = None
+        self,
+        image_query: Union[ImageQuery, str],
+        label: Union[Label, int, str],
+        rois: Union[List[ROI], str, None] = None,
     ):
         """
         Provide a new label (annotation) for an image query. This is used to provide ground-truth labels
         for training detectors, or to correct the results of detectors.
 
         **Example usage**::
 
-            gl = Groundlight()
+            gl = ExperimentalApi()
```

Comment:
Why does this need to be `ExperimentalAPI()`

Reply:
Copy paste, my greatest power and greatest weakness